### ISSUE TITLE
[Adaptive Cards -  "TextBlock" item from "Cards element"]: Incorrect heading structure is defined for the headings present under 'Text Block' screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8798]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer/TextBlock.html
OS Version: 23H2 (OS Build 26020.1000)
Browser Version: Version 122.0.2348.0 (Official build) dev (64-bit)
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL on Edge Dev browser.
2. Navigate to Headings and press enter key to turn on the toggle button 'On',

**Actual Result:**
While navigating to the 'Text Block' screen, incorrect heading structure is defined it is defined as 'H1 > H2 > H3 > H2'.

**Expected Result:**
Heading must be in logical order and Heading structure should be correct and its hierarchy should be as 'H1 > H2 > H3 > H4'. Inherited Properties heading should be defined as H2.

**Note:**
Same issue is repro under 'New Card' screen as 'Create, Explore and other headings' should be defined as H2 instead of H1,

**User Impact:**
Screen reader users will not get the context of the page if they navigate in heading mode as incorrect heading level announcement will be confusing for users as screen reader announces first Heading level 1 and then heading level 3.
 
**WCAG Reference link:** 
[[MAS 1.3.1 – Info and Relationships.docx (sharepoint.com)](https://microsoft.sharepoint.com/:w:/s/accessibility/EXdDUZzldv5MgwDaMHnTzcMBcT8opKGncYjU-gCJ4uqQdg?e=9nuVmt)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)

“ Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section”

### Screenshots

![Incorrect heading order is defined](https://github.com/microsoft/AdaptiveCards/assets/95913748/5000b9cd-e5e5-46eb-9ccc-4f889fe01a4d)

![Incorrect heading level is defined](https://github.com/microsoft/AdaptiveCards/assets/95913748/0196e68c-c1bd-42d3-a23f-c6a49ac12ea6)

### Card JSON

```JSON
NA
```
